pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -994,6 +1007,25 @@ private PdfConfigInfo getPancanReleaseV2PdfConfigIfNeeded(
         return releasePdfConfig;
     }
 
+    private PdfConfigInfo getLmsReleaseV2PdfConfigIfNeeded(
```

Comment:
looked into other pancan special handling to avoid code duplication but activity names, pdf config names being different didn't make it feasible